modules/tinymce/src/core/main/ts/fmt/Preview.ts

```diff
@@ -192,39 +191,40 @@ const getCssText = (editor: Editor, format: any) => {
   }
 
   // Removes any variables since these can't be previewed
-  const removeVars = (val): string => {
-    return val.replace(/%(\w+)/g, '');
+  const removeVars = (val: FormatAttrOrStyleValue): string => {
+    return Type.isString(val) ? val.replace(/%(\w+)/g, '') : '';
```

Comment:
Minor bug fix here technically, as this didn't account for the `val` being a function. As an example, run `editor.formatter.getCssText('lang')` and it'll throw an exception.

Reply:
should we have a test for this? 😉